packages/protocol/test-sol/stability/SortedOracles.t.sol

```diff
@@ -0,0 +1,679 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { Test } from "celo-foundry/Test.sol";
+import { SortedOracles } from "../../contracts/stability/SortedOracles.sol";
+import "../../contracts/common/FixidityLib.sol";
+import "../../contracts/common/linkedlists/AddressSortedLinkedListWithMedian.sol";
+import "../../contracts/common/linkedlists/SortedLinkedListWithMedian.sol";
+
+contract SortedOraclesTest is Test {
+  using FixidityLib for FixidityLib.Fraction;
+  using AddressSortedLinkedListWithMedian for SortedLinkedListWithMedian.List;
+
+  SortedOracles sortedOracle;
+
+  address oracleAccount;
+  address aToken = 0x00000000000000000000000000000000DeaDBeef;
+  uint256 reportExpiry = 3600;
+
+  uint256 constant YEAR_IN_SECONDS = 31536000;
+
+  event OracleAdded(address indexed token, address indexed oracleAddress);
+  event OracleRemoved(address indexed token, address indexed oracleAddress);
+  event OracleReported(
+    address indexed token,
+    address indexed oracle,
+    uint256 timestamp,
+    uint256 value
+  );
+  event OracleReportRemoved(address indexed token, address indexed oracle);
+  event MedianUpdated(address indexed token, uint256 value);
+  event ReportExpirySet(uint256 reportExpiry);
+  event TokenReportExpirySet(address token, uint256 reportExpiry);
+
+  function setUp() public {
+    vm.warp(YEAR_IN_SECONDS);
```

Comment:
why traveling at the beginning? is it to set a baseline?, In that case I then think this is an alternative https://github.com/celo-org/celo-monorepo/pull/10773/files#diff-f7395554b886736da22e519de74623b3087a6c9239a351df77c8586cc6841c07R6

Reply:
Since Foundry tests start at timestamp 0 it makes sense to offset it - I'm using constants now